### ISSUE TITLE
Update copyright dates to 2018

### ISSUE THREAD

**Issue by SD10** [MessageKit/MessageKit#438]
Need to change this in LICENSE.md & all the source files
```diff
-- Copyright (c) 2017 MessageKit
++ Copyright (c) 2017 - 2018 MessageKit
```

**Comment by SD10** [comment]
@richgabrielli Maybe you'd like to take a shot at this one too? There are a lot more dates to change on this one! The `ChatExample` project, the `MessageKit` project, and `LICENSE.md`

**Comment by Sherlouk** [comment]
I mentioned ages ago about defining a copyright template and including it in the project, not sure if someone looked into it before but if it was that'll need updating too!

(If it wasn't, maybe another case to reconsider so this can be the same for new files!)

**Comment by SD10** [comment]
@Sherlouk I’m so pleased that you’re offering to look into this for me!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Sherlouk** [comment]
Tad busy at the moment! But if anyone wants to:
https://oleb.net/blog/2017/07/xcode-9-text-macros/

**Comment by richgabrielli** [comment]
Should I update the year first (brute force) and then have this as another issue that needs to be looked at or combine both?

**Comment by Sherlouk** [comment]
FWIW as well, because I don't want someone brute forcing this. If you just use a simple command line you can replace it all in one swoop! Here's the command:

`find . -name "*.swift" -exec sed -i '' "s/Copyright (c) 2017 MessageKit/Copyright (c) 2017-2018 MessageKit/g" {} \;`

If you run this from the root directory, this will find all the files with the name wildcard .swift (aka all the swift files). It will then execute the provided command with the file name as a parameter.

Using sed we can find and replace the content of the files! It uses "/" as a delimiter so you can see the before and after phases!

Hopefully this'll help next year when you need to do the same thing!

@richgabrielli @SD10

**Comment by richgabrielli** [comment]
I will do this. I will do a few the brute force method so I can peruse some of the files and then research / try out the script. Thanks!

**Comment by richgabrielli** [comment]
Couple questions: I have changed the example project to my info so I could run. Did I mess up the project to submit a pull request?

**Comment by richgabrielli** [comment]
2nd question: found an error in Avatar.playground. Should I just leave it?

<img width="690" alt="screen shot 2018-01-02 at 8 09 19 pm" src="https://user-images.githubusercontent.com/7821160/34506229-e84da822-eff8-11e7-9991-0519969f2c5a.png">

**Comment by SD10** [comment]
@richgabrielli I didn’t even know we had that playground still. You can leave it.

As for building the project, don’t worry about it. We’re not making code changes so I’m not concerned about CI.

If you made some changes to the .xcproj, etc, you can always discard the changes with a `git checkout [file]`



<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by SD10** [comment]
As for the project template that can be a separate issue

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by richgabrielli** [comment]
lol. I will leave the playground. 

I wanted to run the example, so I added my developer ID and did the recommended Xcode changes. Sorry. Should I delete and refork and not mess with it?

**Comment by richgabrielli** [comment]
And I had to run pod update in the Example directory.

**Comment by SD10** [comment]
🤔 You don’t need to refork. Just submit a PR and I’ll take a look. Make sure you check the box that allows me to edit the PR just in case.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by richgabrielli** [comment]
Ready to submit PR. Where do I add you?

**Comment by SD10** [comment]
Closed by #439
